src/OSPSuite.R/Services/DataRepositoryTask.cs

```diff
@@ -28,5 +34,34 @@ public void SaveDataRepository(DataRepository dataRepository, string fileName)
       {
          _pkmlPersistor.SaveToPKML(dataRepository, fileName);
       }
+
+      public DataColumn GetErrorColumn(DataColumn column)
+      {
+         if (column.ContainsRelatedColumn(AuxiliaryType.ArithmeticStdDev))
+            return column.GetRelatedColumn(AuxiliaryType.ArithmeticStdDev);
+
+         if (column.ContainsRelatedColumn(AuxiliaryType.GeometricStdDev))
+            return column.GetRelatedColumn(AuxiliaryType.GeometricStdDev);
+
+         return null;
+      }
+
+      public DataColumn AddErrorColumn(DataColumn column, string name = "yError", string errorType = null)
+      {
+         var errorColumn = new DataColumn(name, column.Dimension, column.BaseGrid);
+
+         if (string.IsNullOrEmpty(errorType))
+            errorType = AuxiliaryType.ArithmeticStdDev.ToString();
+
+         AuxiliaryType auxiliaryType;
+         if (!Enum.TryParse(errorType, out auxiliaryType))
+            throw new Exception(Error.InvalidAuxiliaryType);
```

Comment:
why not `OSPSuiteException`? usually when we throw a (handled) exception, it's always is (or is derived from) the `OSPSuiteException`. The error dialog knows then that exception was handled (otherwise "Oups.." message is shown)
If the function is called from R only it probably does not really matter, but still..

Reply:
Ok, in that case I would say there are a couple of places in the code where I have thrown Exception and not OSPSuiteException (always on the R end). I would try to fix them together on this PR if you do not mind.

Reply:
sure :)

Reply:
Done...